File: app/src/main/java/mobiledev/erickgomes/picpayapp/database/AppDatabase.java
package mobiledev.erickgomes.picpayapp.database;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import mobiledev.erickgomes.picpayapp.models.CreditCard;

/**
 * Created by erickgomes on 24/03/2018.
 */

@Database(entities = {CreditCard.class}, version = 2, exportSchema = false)
public abstract class AppDatabase extends RoomDatabase {
    public abstract CreditCardDao creditCardDao();
}

File: app/src/main/java/br/com/kassianoresende/picpay/model/AppDatabase.kt
package br.com.kassianoresende.picpay.model

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase

@Database(entities = [(CreditCard::class)], version = 1, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {
    abstract fun creditCardDao(): CreditCardDao
}